providers/InProcessFeedProvider.tsx

```
@@ -9,14 +10,27 @@ const InProcessFeedContext = createContext<ReturnType<typeof useFeeds>>(
+            new Date(b.created_at).getTime() - new Date(a.created_at).getTime(),
+        )
+        .filter(
+          (feed) =>
+            !Boolean(
+              hiddenMoments.find(
+                (moment) =>
+                  moment.tokenContract.toLowerCase() ===
+                    feed.collection.toLowerCase() &&
+                  moment.tokenId === feed.tokenId,
+              ),
+            ),
```

Biome (1.9.4)

[error] Avoid redundant `Boolean` call

It is not necessary to use `Boolean` call when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant `Boolean` call


(lint/complexity/noExtraBooleanCast)